aws-athena/views/default-vw_pin_address.sql

```diff
@@ -61,3 +61,5 @@ LEFT JOIN {{ source('iasworld', 'owndat') }} AS own
     AND own.deactivat IS NULL
 WHERE par.cur = 'Y'
     AND par.deactivat IS NULL
+    -- Test parcels have class codes of '999' and need to be removed
+    AND par.class != '999'
```

Comment:
**issue:** We need to check that 999 PINs _only_ represent things we want to exclude from PIN universe. There's a chance that some of these are "used" to do things like fill in the parcel map.